pkgs/tools/nix/info/relaxedsandbox.nix

```diff
@@ -1,12 +1,9 @@
 let
-  pkgs = import <nixpkgs> {};
-in pkgs.runCommand "diagnostics-sandbox"
-  {
-    __noChroot = true;
-  }
-  ''
-    set -x
-    # no cache: ${toString builtins.currentTime}
-    test -d "$(dirname "$out")/../var/nix"
-    touch $out
-  ''
+  pkgs = import <nixpkgs> { };
+in
+pkgs.runCommand "diagnostics-sandbox" { __noChroot = true; } ''
```

Comment:
Do we really need to collapse this attrset into a single line format?
This will add a bunch of diff noise when new attrset are added, won't it?

Reply:
I'm not sure if we'll go for that in the end, but with https://github.com/NixOS/nixfmt/pull/222 (which also fixes your other complaint), this file would look like this instead:

```nix
let
  pkgs = import <nixpkgs> { };
in
pkgs.runCommand "diagnostics-sandbox"
  {
    __noChroot = true;
  }
  ''
    set -x
    # no cache: ${toString builtins.currentTime}
    test -d "$(dirname "$out")/../var/nix"
    touch $out
  ''
```